### ISSUE TITLE
Neofetch not displaying image

### ISSUE THREAD

**Issue by AlexanderSchmedes** [dylanaraps/neofetch#513]
## Description

Neofetch doesn't display the wallpaper and falls back to ascii. I'm running it in iTerm2.

#### Neofetch version

Neofetch Version is stable 2.0.2.

## Verbose log

Log: https://gist.github.com/AlexanderSchmedes/bbd2433d87acd68f3ae5eaed442a6dbe

**Comment by dylanaraps** [comment]
Fixed, thanks for reporting this.

**Comment by AlexanderSchmedes** [comment]
Hey, can't verify that it works, right now, since I have to wait till this gets updated on homebrew but thank you very much for the fast fix!

**Comment by dylanaraps** [comment]
No problem!

The bug was that we were checking if `w3m-img` existed on all Operating Systems instead of just systems running an X server. When the check failed on OS X, Neofetch disabled image mode and just displayed ascii art.

The fix was to just move the `w3m-img` check so that it's run only on the Operating Systems we want to run it on. This fix only moves a couple of lines around so if you can't wait until next release you can just patch your Neofetch.

Here's the patch file for the fix: **[patch](https://github.com/dylanaraps/neofetch/commit/926dea972b82d1b81e5501e63c8d4395ee274b84.patch)**

**Comment by AlexanderSchmedes** [comment]
Yea, I read the commit, didn't think about how to patch it manually though. I patched it now by replacing my neofetch file with the new one, but I get a different error now.

Log: https://gist.github.com/AlexanderSchmedes/7705aeec3e19588cd6cb5990e36c2a4e

**Comment by dylanaraps** [comment]
Neofetch can't find an image file, does this command work?

```sh
# Replace this with a path to an image.
neofetch --image ~/path/to/img.png

# Also try this command.
neofetch --image wall
```

One more thing, 2.0.1 made ascii mode the default. This change was made to make Neofetch easier to use for new users since ascii mode is the most used mode.

See this section of the image wiki page:

https://github.com/dylanaraps/neofetch/wiki/Images-in-the-terminal#enabling-image-mode

**Comment by AlexanderSchmedes** [comment]
Hey, didn't know that ascii is the default now. neofetch --image wall worked immediatly.  After that I looked at the config file for a bit. I changed it now to image_source = wallpaper and it works perfectly, thanks! In the config the default entry is image_source = 'ascii', or "ascii" on the here uploaded config file, but I think the '/" should be removed since they are the reason I got the error message "Image: '‘ascii’' doesn't exist, falling back to ascii mode.".
